### PR TITLE
Azure fix [DRAFT]

### PR DESCRIPTION
**Pull request by adrimbarean**
Small fixes to help with azure checks.

### CODE REVIEW

File: drivers/adc/ad7124/iio_ad7124.c [hunk 1]
```diff
@@ -475,6 +475,9 @@ static ssize_t ad7124_iio_change_scale_chan(void *device, char *buf, size_t len,
 static bool get_next_ch_idx(uint32_t ch_mask, uint32_t last_idx,
 			    uint32_t *new_idx)
 {
+	if (last_idx > 32)
+		return 0;
```
Comment (amiclaus) [inline drivers/adc/ad7124/iio_ad7124.c:479]: maybe use defines here

**Review by amiclaus — APPROVED** [review]
Minor comment. The rest LGTM.

**Comment by adrimbarean** [comment]
V1:
 - truly fix inclusion problem in driver ad9081;
 - fix astyle.

**Comment by adrimbarean** [comment]
V2:
 - use 'false' instead of 0 in the ad7124 fix

**Pull request merged**